docs/docs/principles.md

```
@@ -0,0 +1,33 @@
+# Community Principles
+
+### **Inclusive and Diversity**
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)